cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -91,5 +94,20 @@ func updateAllInstanceAccessConfig(workflow *daisy.Workflow, accessConfigProvide
 			}
 		}
 	}
+}
+
+// RemovePrivacyLogInfo removes privacy log information.
+func RemovePrivacyLogInfo(message string) string {
+	// Since translation scripts are vary and is hard to predict the output, we have to hide the
+	// details and only remain "TranslateFailed"
+	if strings.Contains(message, "TranslateFailed") {
+		return "TranslateFailed"
+	}
+
+	// All import/export bash scripts bracks privacy info inside "[Privacy-> XXX <-Privacy]". Let's
+	// remove it for privacy.
+	var re = regexp.MustCompile(`\[Privacy\->.*?<\-Privacy\]`)
```

Comment:
If using MustCompile make this a global var so the panic happens immediately an not at function run time.

Reply:
Right, that is by design. We use this regex to detect privacy info. It shouldn't fail due that it's a static expression.

Reply:
But it should be a global var so failure happens immediately on binary execution